pkg/vm/builtins/modulo.go

```diff
@@ -320,27 +321,32 @@ func (m *ModBuiltin) writeNWordsValue(mem *memory.Memory, addr memory.MemoryAddr
 // Returns true on success or if all values are already known.
 // Given known, res, p fillValue tries to compute the minimal integer operand x which
 // satisfies the equation op(x,known) = res + k*p for some k in {0,1,...,self.k_bound-1}.
-func (m *ModBuiltin) fillValue(mem *memory.Memory, inputs ModBuiltinInputs, index int, op ModBuiltinType) (bool, error) {
+func (m *ModBuiltin) fillValue(mem *memory.Memory, inputs ModBuiltinInputs, index int, op ModBuiltinType) (int, error) {
 	addresses := make([]memory.MemoryAddress, 0, 3)
 	values := make([]*big.Int, 0, 3)
 
 	for i := 0; i < 3; i++ {
 		addr, err := inputs.offsetsPtr.AddOffset(int16(3*index + i))
 		if err != nil {
-			return false, err
+			return 0, err
 		}
 		offsetFelt, err := mem.ReadAsElement(addr.SegmentIndex, addr.Offset)
 		if err != nil {
-			return false, err
+			return 0, err
 		}
 		offset := offsetFelt.Uint64()
 		addr, err = inputs.valuesPtr.AddOffset(int16(offset))
 		if err != nil {
-			return false, err
+			return 0, err
 		}
 		addresses = append(addresses, addr)
 		// do not check for error, as the value might not be in memory
```

Comment:
This comment should be removed now right?

Reply:
not exactly. Modified it.